.github/workflows/build.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         run: git lfs pull
       - name: Build with make
         run: make
+      - name: Send coverage report to Codacy
+        env:
+          CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        if: success()
+        run: bash <(curl -Ls https://coverage.codacy.com/get.sh) report -r assets/build/reports/jacoco/test/jacocoTestReport.xml
```

Comment:
maybe you could add this also to the build-windows branch below

Reply:
Sure, done! I wasn't sure if the bash command would have worked.

Reply:
you're right, it's not working, I didn't realized that, sorry,
I just saw the windows build passing and wondered why